lua/lib/fs.lua

```diff
@@ -54,6 +54,7 @@ function M.create(node)
   if not ans or #ans == 0 then return end
 
   if not ans:match('/') then
+    node.open = true
```

Comment:
not too sure what this change implies ?

Reply:
So the reason this exists is so that an empty folder does not stay closed when a new item is added. Since there are only two ways a folder updates its "fullness" icon (when first being added to the tree, an upon being expanded), this forces a folder to update when a new item is added.

I could possibly replace this with setting the `node.children_initial` or `has_children` variables again, but I thought this was nice, and if we do switch it out then I might make a PR with just this auto expand feature 😂

Reply:
allright keep it like this, it would indeed be better this way